lib/nacre/channel.rb

```diff
@@ -27,20 +27,12 @@ def warehouse_ids=(hash)
 
     private
 
-    def self.request_options
-      ''
-    end
-
     def self.service_name
-      'channel'
+      'product'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.